apis/python/src/tiledbsoma/_query.py

```diff
@@ -457,13 +464,21 @@ def to_anndata(
         var_joinids = self.var_joinids()
 
         x_matrices = {
-            _xname: tp.submit(
-                _read_as_csr,
-                layer,
-                obs_joinids,
-                var_joinids,
-                self._indexer.by_obs,
-                self._indexer.by_var,
+            _xname: (
+                tp.submit(
+                    _read_as_csr,
+                    layer,
+                    obs_joinids,
+                    var_joinids,
+                    self._indexer.by_obs,
+                    self._indexer.by_var,
+                )
+                if not dask
+                else load_daskarray(
```

Comment:
naming nit: should private functions have a leading underscore?  (I'm not sure. but the rest of this module follows that pattern)

Reply:
My thought was we're under `tiledbsoma._dask`, so things are private by default / as far as external users are concerned. 

otoh, `load_daskarray` is used from `_query`, `_sparse_nd_array`, and `io.outgest`, so it feels sort of public within the project.

I'm fine to change it, or follow whatever convention you prefer.